src/material/sort/sort.ts

```diff
@@ -107,6 +118,10 @@ export class MatSort extends _MatSortMixinBase
   /** Event emitted when the user changes either the active sort or sort direction. */
   @Output('matSortChange') readonly sortChange: EventEmitter<Sort> = new EventEmitter<Sort>();
 
+  constructor(@Inject(MAT_SORT_DEFAULT_OPTIONS) private readonly _defaultOptions: MatSortDefaultOptions) {
```

Comment:
This should be marked as an optional parameter so that it doesn't break existing cases that extend `MatSort`. I'm guessing that we want `@Optional` on it too since the token might not be provided. Finally, I think that the `readonly` here isn't necessary since it's a private property already.

Reply:
sgtm, fixed